crux-kafka/src/crux/kafka.clj

```diff
@@ -152,33 +158,23 @@
                                        (TopicPartition. (.topic r)
                                                         (.partition r))) records)
         partitions (vec (keys partition->records))
-        end-offsets (.endOffsets consumer partitions)
         stored-consumer-state (or (db/read-index-meta indexer :crux.tx-log/consumer-state) {})
         consumer-state (->> (for [^TopicPartition partition partitions
                                   :let [^ConsumerRecord last-record-in-batch (->> (get partition->records partition)
                                                                                   (sort-by #(.offset ^ConsumerRecord %))
                                                                                   (last))
-                                        next-offset (inc (.offset last-record-in-batch))
-                                        end-offset (get end-offsets partition)
-                                        lag (- end-offset next-offset)]]
-                              (do (when-not (zero? lag)
-                                    (log/debug "Falling behind" (str partition) "at:" next-offset "end:" end-offset))
-                                  [(topic-partition-meta-key partition)
-                                   {:next-offset next-offset
-                                    :time (Date. (.timestamp last-record-in-batch))
-                                    :lag lag}]))
+                                        next-offset (inc (.offset last-record-in-batch))]]
```

Comment:
Previously this lag was show in the status map. I know we discussed that this would become a calculated property, is the latest known offset also exposed into the status map somehow? Or is this "broken" until the new monitoring work catches up?

Reply:
yeah - I'm going to hold fire on reinstating this til metrics come in, unless there's something depending on it that'll completely break if it's not provided?